components/script/webdriver_handlers.rs

```diff
@@ -212,7 +223,7 @@ pub(crate) unsafe fn jsval_to_webdriver(
         });
         let _ac = JSAutoRealm::new(cx, *object);
 
-        if is_array_like(cx, val) {
+        if is_array_like(cx, val) || is_arguments_object(cx, val) {
```

Comment:
Is "array-like" defined in some spec? Would it make more sense to include arguments objects there instead of checking both conditions here?

Reply:
I wanted to avoid this because is_array_like is also used in webidl codegen for determining overload resolution. I think there's an opportunity to remove it in the future (the equivalent gecko code looks like it's had a lot of updates) but I'd rather not tie that work into this PR.